examples/partial_execution/README.md

```diff
@@ -0,0 +1,34 @@
+# Partial execution
+
+The structure of the example is visualized in [pipeline.svg](./pipeline.svg).
```

Comment:
Include the image not a link, see https://stackoverflow.com/a/16462143 for instructions